app/src/main/java/de/danoeh/antennapod/fragment/swipeactions/SwipeActions.java

```diff
@@ -132,6 +132,16 @@ public boolean onMove(@NonNull RecyclerView recyclerView,
 
     @Override
     public void onSwiped(@NonNull RecyclerView.ViewHolder viewHolder, int swipeDir) {
+
+        if (tag.equals("PlaybackHistoryFragment")) {
```

Comment:
This is still quite weird. Then SwipeActions has an implicit dependency on the Fragment it is shown in. Instead, I would add a normal Swipe Action selection dialog like on the other screens, with "remove from history" as the default actions. Then this class here does not need to be changed.